src/guides/wiki/oba-2x-config-deploy-guide.md

```diff
@@ -5,42 +5,44 @@ layout: page
 
 <div class='bg-blue-50 border-blue-500 dark:bg-slate-800 p-4 rounded-md'>
```

Comment:
I think you should remove this top `<div>`. I added this to the guide in order to clarify how out of date it was, and your changes fix that! Once this gets merged and deployed, I'll update the wiki to point to this guide as the new canonical source.